src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int LIMIT = 6;
+    private static final int LIMIT_SIDE_SHAPES = 10;
+    private ColorSupplier color = new ColorSupplier();
+
+    public Shape[] getRandomFigure() {
```

Comment:
всередині у тебе створюється одна фігура, а не масив
```suggestion
    public Shape getRandomFigure() {
```